src/js/components/Notification/Notification.js

```diff
@@ -134,16 +137,19 @@ const Notification = ({
   let message = messageProp;
 
   if (actionsProp)
-    actions = actionsProp.map((action, index) => (
-      <NotificationAnchor
-        key={action.label}
-        // create space between first anchor and text content
-        margin={
-          index === 0 && (message || title) ? { left: 'xsmall' } : undefined
-        }
-        {...action}
-      />
-    ));
+    actions = actionsProp.map((action, index) => {
+      let margin;
+      if (index === 0 && (message || title)) margin = { left: 'xsmall' };
+      else if (index > 0) margin = { left: 'xsmall' };
```

Comment:
Can we add two anchors in the 'actions' Notifications test to make sure this condition is captured in the snapshot?

Reply:
yes